src/components/Footer/Footer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import './Footer.css';
+
+const Footer = (props) => {
+  return (
+
+    <footer>
+      <Link to='/about'>About</Link>
```

Comment:
About page will explain context for project + how the duck makes decisions about which advice to give